Izzy git branch

izzygaw opened the pull request.

izzygaw committed:
Add script to make xx plots

izzygaw committed:
Merge branch 'master' of https://github.com/Rbootcamp-UHM/Iris-repo into IzzyGitBranch

izzygaw committed:
dogs

izzygaw committed:
2puppies

izzygaw committed:
3puppies

izzygaw committed:
6 puppies

izzygaw committed:
updated puppy

mbutler808 commented on HW1IzzyGaw.R:
```diff
@@ -0,0 +1,17 @@
+iris #Looking at the whole data set
+SepL <- iris$Sepal.Length #Naming a column something easier to grab. Also taking Sepal Length data out of the larger data set, so I can impliment it elsewhere
+Sp <- iris$Species #same as above.
+plot(Sp, SepL) #Here I am making sure that plot works and seeing what the data looks like. I thought it would be most interesting to look at a character vs. a numeric value. Especially when looking at biological data, it is fun to look at morphological differences across species
```
This is great! But so hard to read. Could you please add some white space?

mbutler808 commented on HW1IzzyGaw.R:
```diff
@@ -0,0 +1,17 @@
+plot(Sp, SepL, cex=2, pch=19, col="maroon") #Here I am adding to the plot above. I added color to the plot, chose a style for the one outlier point, and increased the size of the points. This part is different than the examples given in iris stub because only just color was an added feature to the example plots. I also tried to add a fun color. 
+plot(Sp, SepL, xlab= "Sepal Length", ylab= "Species",cex=2, pch=19, col="maroon") #Here I add x and y labels. Deviating even further from the iris stub example. I am trying to replicate a plot that I would use in my research. In other words, I am testing out R skills/coding that I can impliment later in my academic career. Lastly, we had not tried a box plot yet and I was eager to try.
+plot(iris$Species, iris$Sepal.Length, main="Plants Are Interesting", xlab= "Sepal Length", ylab= "Species",cex=2, pch=19, col="maroon") #Here I add a main title as well. 
+plot(Sp, SepL, main="Plants Are Intruiging", ylab= "Sepal Length", xlab= "Species",cex=2, pch=19, col="maroon") #Here I am seeing if I can make the same plot as above but in a more concise way. It worked!
```
Yay!! Excellent. Be brave!

mbutler808 commented on HW1IzzyGaw.R:
```diff
@@ -0,0 +1,17 @@
+SepL <- iris$Sepal.Length #Naming a column something easier to grab. Also taking Sepal Length data out of the larger data set, so I can impliment it elsewhere
+Sp <- iris$Species #same as above.
+plot(Sp, SepL) #Here I am making sure that plot works and seeing what the data looks like. I thought it would be most interesting to look at a character vs. a numeric value. Especially when looking at biological data, it is fun to look at morphological differences across species
+plot(Sp, SepL, cex=2, pch=19, col="maroon") #Here I am adding to the plot above. I added color to the plot, chose a style for the one outlier point, and increased the size of the points. This part is different than the examples given in iris stub because only just color was an added feature to the example plots. I also tried to add a fun color. 
```
I love the idea of indicating the outlier, but I am not seeing how you do it.

mbutler808 commented on Markdown_HW1_IzzyGaw.Rmd:
```diff
@@ -0,0 +1,20 @@
+output: html_document
+---
+
+This is a text paragraph in an R Markdown document.
```
Great job inserting your R code below. However, can you please replace the example text with your explanations?
Remember, effective documentation is concise :)

mbutler808 commented on Markdown_HW1_IzzyGaw.Rmd:
```diff
@@ -0,0 +1,20 @@
+
+This is a text paragraph in an R Markdown document.
+
+Below is working R code in a code chunk: 
```
Just replace these lines with something informative

mbutler808 reviewed (changes requested):
Very nice Izzy!!  Just a few comments. Could you please make your R script more readable? And could you please replace the template text with your narrative in the .Rmd file? (and also render to .html) thanks!

After you fix it I can push it to the code website. 
Marguerite

izzygaw commented:
Hello!

Sounds good! I can do all of that.

Are you saying that you aren't seeing the .html file or to re-render it
once I fix the .rmd? If you aren't seeing it, I promise I uploaded it but I
did do it first, so it might be under a different commit -m name, 3 puppies
I think. Let me know! Thanks for being so patient with all my questions!

Please let me know if you see it! I will fix everything! Thank you so much
for your help and comments.

All the best,
Izzy

On Sat, Jan 18, 2020 at 2:04 PM mbutler808 <notifications@github.com> wrote:

> *@mbutler808* requested changes on this pull request.
>
> Very nice Izzy!! Just a few comments. Could you please make your R script
> more readable? And could you please replace the template text with your
> narrative in the .Rmd file? (and also render to .html) thanks!
>
> After you fix it I can push it to the code website.
> Marguerite
> ------------------------------
>
> In HW1IzzyGaw.R
> <https://github.com/Rbootcamp-UHM/Iris-repo/pull/13#discussion_r368255426>
> :
>
> > +iris #Looking at the whole data set
> +SepL <- iris$Sepal.Length #Naming a column something easier to grab. Also taking Sepal Length data out of the larger data set, so I can impliment it elsewhere
> +Sp <- iris$Species #same as above.
> +plot(Sp, SepL) #Here I am making sure that plot works and seeing what the data looks like. I thought it would be most interesting to look at a character vs. a numeric value. Especially when looking at biological data, it is fun to look at morphological differences across species
>
> This is great! But so hard to read. Could you please add some white space?
> ------------------------------
>
> In HW1IzzyGaw.R
> <https://github.com/Rbootcamp-UHM/Iris-repo/pull/13#discussion_r368255512>
> :
>
> > @@ -0,0 +1,17 @@
> +iris #Looking at the whole data set
> +SepL <- iris$Sepal.Length #Naming a column something easier to grab. Also taking Sepal Length data out of the larger data set, so I can impliment it elsewhere
> +Sp <- iris$Species #same as above.
> +plot(Sp, SepL) #Here I am making sure that plot works and seeing what the data looks like. I thought it would be most interesting to look at a character vs. a numeric value. Especially when looking at biological data, it is fun to look at morphological differences across species
> +plot(Sp, SepL, cex=2, pch=19, col="maroon") #Here I am adding to the plot above. I added color to the plot, chose a style for the one outlier point, and increased the size of the points. This part is different than the examples given in iris stub because only just color was an added feature to the example plots. I also tried to add a fun color.
> +plot(Sp, SepL, xlab= "Sepal Length", ylab= "Species",cex=2, pch=19, col="maroon") #Here I add x and y labels. Deviating even further from the iris stub example. I am trying to replicate a plot that I would use in my research. In other words, I am testing out R skills/coding that I can impliment later in my academic career. Lastly, we had not tried a box plot yet and I was eager to try.
> +plot(iris$Species, iris$Sepal.Length, main="Plants Are Interesting", xlab= "Sepal Length", ylab= "Species",cex=2, pch=19, col="maroon") #Here I add a main title as well.
> +plot(Sp, SepL, main="Plants Are Intruiging", ylab= "Sepal Length", xlab= "Species",cex=2, pch=19, col="maroon") #Here I am seeing if I can make the same plot as above but in a more concise way. It worked!
>
> Yay!! Excellent. Be brave!
> ------------------------------
>
> In HW1IzzyGaw.R
> <https://github.com/Rbootcamp-UHM/Iris-repo/pull/13#discussion_r368255564>
> :
>
> > @@ -0,0 +1,17 @@
> +iris #Looking at the whole data set
> +SepL <- iris$Sepal.Length #Naming a column something easier to grab. Also taking Sepal Length data out of the larger data set, so I can impliment it elsewhere
> +Sp <- iris$Species #same as above.
> +plot(Sp, SepL) #Here I am making sure that plot works and seeing what the data looks like. I thought it would be most interesting to look at a character vs. a numeric value. Especially when looking at biological data, it is fun to look at morphological differences across species
> +plot(Sp, SepL, cex=2, pch=19, col="maroon") #Here I am adding to the plot above. I added color to the plot, chose a style for the one outlier point, and increased the size of the points. This part is different than the examples given in iris stub because only just color was an added feature to the example plots. I also tried to add a fun color.
>
> I love the idea of indicating the outlier, but I am not seeing how you do
> it.
> ------------------------------
>
> In Markdown_HW1_IzzyGaw.Rmd
> <https://github.com/Rbootcamp-UHM/Iris-repo/pull/13#discussion_r368255608>
> :
>
> > @@ -0,0 +1,20 @@
> +---
> +title: "R Markdown Super Awesome Homework 1"
> +author: "Izzy Gaw"
> +date: "2020-01-16"
> +output: html_document
> +---
> +
> +This is a text paragraph in an R Markdown document.
>
> Great job inserting your R code below. However, can you please replace the
> example text with your explanations?
> Remember, effective documentation is concise :)
> ------------------------------
>
> In Markdown_HW1_IzzyGaw.Rmd
> <https://github.com/Rbootcamp-UHM/Iris-repo/pull/13#discussion_r368255615>
> :
>
> > @@ -0,0 +1,20 @@
> +---
> +title: "R Markdown Super Awesome Homework 1"
> +author: "Izzy Gaw"
> +date: "2020-01-16"
> +output: html_document
> +---
> +
> +This is a text paragraph in an R Markdown document.
> +
> +Below is working R code in a code chunk:
>
> Just replace these lines with something informative
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Rbootcamp-UHM/Iris-repo/pull/13?email_source=notifications&email_token=AOI5PFU2IE5XQJEE6RED2T3Q6OKIVA5CNFSM4KIVXKRKYY3PNVWWK3TUL52HS4DFWFIHK3DMKJSXC5LFON2FEZLWNFSXPKTDN5WW2ZLOORPWSZGOCSH4YDQ#pullrequestreview-344968206>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AOI5PFS64FODZ33CNKKVKALQ6OKIVANCNFSM4KIVXKRA>
> .
>


-- 
*Isabella M. Gaw*
*Marine Biology **Master's Student*
*Cole Lab, **Department of Biology*
*University of Hawai'i at Mānoa*

mbutler808 commented:
I can see it. Don’t worry. I meant generate it again after fixing. 

Look on the GitHub website as well. The GUI interface for changes and comments is nice. just FYI. 

Marguerite 

Sent from my iPhone

> On Jan 18, 2020, at 2:20 PM, izzygaw <notifications@github.com> wrote:
> 
> ﻿Hello!
> 
> Sounds good! I can do all of that.
> 
> Are you saying that you aren't seeing the .html file or to re-render it
> once I fix the .rmd? If you aren't seeing it, I promise I uploaded it but I
> did do it first, so it might be under a different commit -m name, 3 puppies
> I think. Let me know! Thanks for being so patient with all my questions!
> 
> Please let me know if you see it! I will fix everything! Thank you so much
> for your help and comments.
> 
> All the best,
> Izzy
> 
> On Sat, Jan 18, 2020 at 2:04 PM mbutler808 <notifications@github.com> wrote:
> 
> > *@mbutler808* requested changes on this pull request.
> >
> > Very nice Izzy!! Just a few comments. Could you please make your R script
> > more readable? And could you please replace the template text with your
> > narrative in the .Rmd file? (and also render to .html) thanks!
> >
> > After you fix it I can push it to the code website.
> > Marguerite
> > ------------------------------
> >
> > In HW1IzzyGaw.R
> > <https://github.com/Rbootcamp-UHM/Iris-repo/pull/13#discussion_r368255426>
> > :
> >
> > > +iris #Looking at the whole data set
> > +SepL <- iris$Sepal.Length #Naming a column something easier to grab. Also taking Sepal Length data out of the larger data set, so I can impliment it elsewhere
> > +Sp <- iris$Species #same as above.
> > +plot(Sp, SepL) #Here I am making sure that plot works and seeing what the data looks like. I thought it would be most interesting to look at a character vs. a numeric value. Especially when looking at biological data, it is fun to look at morphological differences across species
> >
> > This is great! But so hard to read. Could you please add some white space?
> > ------------------------------
> >
> > In HW1IzzyGaw.R
> > <https://github.com/Rbootcamp-UHM/Iris-repo/pull/13#discussion_r368255512>
> > :
> >
> > > @@ -0,0 +1,17 @@
> > +iris #Looking at the whole data set
> > +SepL <- iris$Sepal.Length #Naming a column something easier to grab. Also taking Sepal Length data out of the larger data set, so I can impliment it elsewhere
> > +Sp <- iris$Species #same as above.
> > +plot(Sp, SepL) #Here I am making sure that plot works and seeing what the data looks like. I thought it would be most interesting to look at a character vs. a numeric value. Especially when looking at biological data, it is fun to look at morphological differences across species
> > +plot(Sp, SepL, cex=2, pch=19, col="maroon") #Here I am adding to the plot above. I added color to the plot, chose a style for the one outlier point, and increased the size of the points. This part is different than the examples given in iris stub because only just color was an added feature to the example plots. I also tried to add a fun color.
> > +plot(Sp, SepL, xlab= "Sepal Length", ylab= "Species",cex=2, pch=19, col="maroon") #Here I add x and y labels. Deviating even further from the iris stub example. I am trying to replicate a plot that I would use in my research. In other words, I am testing out R skills/coding that I can impliment later in my academic career. Lastly, we had not tried a box plot yet and I was eager to try.
> > +plot(iris$Species, iris$Sepal.Length, main="Plants Are Interesting", xlab= "Sepal Length", ylab= "Species",cex=2, pch=19, col="maroon") #Here I add a main title as well.
> > +plot(Sp, SepL, main="Plants Are Intruiging", ylab= "Sepal Length", xlab= "Species",cex=2, pch=19, col="maroon") #Here I am seeing if I can make the same plot as above but in a more concise way. It worked!
> >
> > Yay!! Excellent. Be brave!
> > ------------------------------
> >
> > In HW1IzzyGaw.R
> > <https://github.com/Rbootcamp-UHM/Iris-repo/pull/13#discussion_r368255564>
> > :
> >
> > > @@ -0,0 +1,17 @@
> > +iris #Looking at the whole data set
> > +SepL <- iris$Sepal.Length #Naming a column something easier to grab. Also taking Sepal Length data out of the larger data set, so I can impliment it elsewhere
> > +Sp <- iris$Species #same as above.
> > +plot(Sp, SepL) #Here I am making sure that plot works and seeing what the data looks like. I thought it would be most interesting to look at a character vs. a numeric value. Especially when looking at biological data, it is fun to look at morphological differences across species
> > +plot(Sp, SepL, cex=2, pch=19, col="maroon") #Here I am adding to the plot above. I added color to the plot, chose a style for the one outlier point, and increased the size of the points. This part is different than the examples given in iris stub because only just color was an added feature to the example plots. I also tried to add a fun color.
> >
> > I love the idea of indicating the outlier, but I am not seeing how you do
> > it.
> > ------------------------------
> >
> > In Markdown_HW1_IzzyGaw.Rmd
> > <https://github.com/Rbootcamp-UHM/Iris-repo/pull/13#discussion_r368255608>
> > :
> >
> > > @@ -0,0 +1,20 @@
> > +---
> > +title: "R Markdown Super Awesome Homework 1"
> > +author: "Izzy Gaw"
> > +date: "2020-01-16"
> > +output: html_document
> > +---
> > +
> > +This is a text paragraph in an R Markdown document.
> >
> > Great job inserting your R code below. However, can you please replace the
> > example text with your explanations?
> > Remember, effective documentation is concise :)
> > ------------------------------
> >
> > In Markdown_HW1_IzzyGaw.Rmd
> > <https://github.com/Rbootcamp-UHM/Iris-repo/pull/13#discussion_r368255615>
> > :
> >
> > > @@ -0,0 +1,20 @@
> > +---
> > +title: "R Markdown Super Awesome Homework 1"
> > +author: "Izzy Gaw"
> > +date: "2020-01-16"
> > +output: html_document
> > +---
> > +
> > +This is a text paragraph in an R Markdown document.
> > +
> > +Below is working R code in a code chunk:
> >
> > Just replace these lines with something informative
> >
> > —
> > You are receiving this because you authored the thread.
> > Reply to this email directly, view it on GitHub
> > <https://github.com/Rbootcamp-UHM/Iris-repo/pull/13?email_source=notifications&email_token=AOI5PFU2IE5XQJEE6RED2T3Q6OKIVA5CNFSM4KIVXKRKYY3PNVWWK3TUL52HS4DFWFIHK3DMKJSXC5LFON2FEZLWNFSXPKTDN5WW2ZLOORPWSZGOCSH4YDQ#pullrequestreview-344968206>,
> > or unsubscribe
> > <https://github.com/notifications/unsubscribe-auth/AOI5PFS64FODZ33CNKKVKALQ6OKIVANCNFSM4KIVXKRA>
> > .
> >
> 
> 
> -- 
> *Isabella M. Gaw*
> *Marine Biology **Master's Student*
> *Cole Lab, **Department of Biology*
> *University of Hawai'i at Mānoa*
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or unsubscribe.

izzygaw committed:
updated puppies

izzygaw commented:
Hello,

Awesome! Thanks for the clarification.

It's all done! I hope it's better, please let me know if there is
anything else I could do to make it *even* better.

All the best,
Izzy

On Sat, Jan 18, 2020 at 3:53 PM mbutler808 <notifications@github.com> wrote:

> I can see it. Don’t worry. I meant generate it again after fixing.
>
> Look on the GitHub website as well. The GUI interface for changes and
> comments is nice. just FYI.
>
> Marguerite
>
> Sent from my iPhone
>
> > On Jan 18, 2020, at 2:20 PM, izzygaw <notifications@github.com> wrote:
> >
> > ﻿Hello!
> >
> > Sounds good! I can do all of that.
> >
> > Are you saying that you aren't seeing the .html file or to re-render it
> > once I fix the .rmd? If you aren't seeing it, I promise I uploaded it
> but I
> > did do it first, so it might be under a different commit -m name, 3
> puppies
> > I think. Let me know! Thanks for being so patient with all my questions!
> >
> > Please let me know if you see it! I will fix everything! Thank you so
> much
> > for your help and comments.
> >
> > All the best,
> > Izzy
> >
> > On Sat, Jan 18, 2020 at 2:04 PM mbutler808 <notifications@github.com>
> wrote:
> >
> > > *@mbutler808* requested changes on this pull request.
> > >
> > > Very nice Izzy!! Just a few comments. Could you please make your R
> script
> > > more readable? And could you please replace the template text with your
> > > narrative in the .Rmd file? (and also render to .html) thanks!
> > >
> > > After you fix it I can push it to the code website.
> > > Marguerite
> > > ------------------------------
> > >
> > > In HW1IzzyGaw.R
> > > <
> https://github.com/Rbootcamp-UHM/Iris-repo/pull/13#discussion_r368255426>
> > > :
> > >
> > > > +iris #Looking at the whole data set
> > > +SepL <- iris$Sepal.Length #Naming a column something easier to grab.
> Also taking Sepal Length data out of the larger data set, so I can
> impliment it elsewhere
> > > +Sp <- iris$Species #same as above.
> > > +plot(Sp, SepL) #Here I am making sure that plot works and seeing what
> the data looks like. I thought it would be most interesting to look at a
> character vs. a numeric value. Especially when looking at biological data,
> it is fun to look at morphological differences across species
> > >
> > > This is great! But so hard to read. Could you please add some white
> space?
> > > ------------------------------
> > >
> > > In HW1IzzyGaw.R
> > > <
> https://github.com/Rbootcamp-UHM/Iris-repo/pull/13#discussion_r368255512>
> > > :
> > >
> > > > @@ -0,0 +1,17 @@
> > > +iris #Looking at the whole data set
> > > +SepL <- iris$Sepal.Length #Naming a column something easier to grab.
> Also taking Sepal Length data out of the larger data set, so I can
> impliment it elsewhere
> > > +Sp <- iris$Species #same as above.
> > > +plot(Sp, SepL) #Here I am making sure that plot works and seeing what
> the data looks like. I thought it would be most interesting to look at a
> character vs. a numeric value. Especially when looking at biological data,
> it is fun to look at morphological differences across species
> > > +plot(Sp, SepL, cex=2, pch=19, col="maroon") #Here I am adding to the
> plot above. I added color to the plot, chose a style for the one outlier
> point, and increased the size of the points. This part is different than
> the examples given in iris stub because only just color was an added
> feature to the example plots. I also tried to add a fun color.
> > > +plot(Sp, SepL, xlab= "Sepal Length", ylab= "Species",cex=2, pch=19,
> col="maroon") #Here I add x and y labels. Deviating even further from the
> iris stub example. I am trying to replicate a plot that I would use in my
> research. In other words, I am testing out R skills/coding that I can
> impliment later in my academic career. Lastly, we had not tried a box plot
> yet and I was eager to try.
> > > +plot(iris$Species, iris$Sepal.Length, main="Plants Are Interesting",
> xlab= "Sepal Length", ylab= "Species",cex=2, pch=19, col="maroon") #Here I
> add a main title as well.
> > > +plot(Sp, SepL, main="Plants Are Intruiging", ylab= "Sepal Length",
> xlab= "Species",cex=2, pch=19, col="maroon") #Here I am seeing if I can
> make the same plot as above but in a more concise way. It worked!
> > >
> > > Yay!! Excellent. Be brave!
> > > ------------------------------
> > >
> > > In HW1IzzyGaw.R
> > > <
> https://github.com/Rbootcamp-UHM/Iris-repo/pull/13#discussion_r368255564>
> > > :
> > >
> > > > @@ -0,0 +1,17 @@
> > > +iris #Looking at the whole data set
> > > +SepL <- iris$Sepal.Length #Naming a column something easier to grab.
> Also taking Sepal Length data out of the larger data set, so I can
> impliment it elsewhere
> > > +Sp <- iris$Species #same as above.
> > > +plot(Sp, SepL) #Here I am making sure that plot works and seeing what
> the data looks like. I thought it would be most interesting to look at a
> character vs. a numeric value. Especially when looking at biological data,
> it is fun to look at morphological differences across species
> > > +plot(Sp, SepL, cex=2, pch=19, col="maroon") #Here I am adding to the
> plot above. I added color to the plot, chose a style for the one outlier
> point, and increased the size of the points. This part is different than
> the examples given in iris stub because only just color was an added
> feature to the example plots. I also tried to add a fun color.
> > >
> > > I love the idea of indicating the outlier, but I am not seeing how you
> do
> > > it.
> > > ------------------------------
> > >
> > > In Markdown_HW1_IzzyGaw.Rmd
> > > <
> https://github.com/Rbootcamp-UHM/Iris-repo/pull/13#discussion_r368255608>
> > > :
> > >
> > > > @@ -0,0 +1,20 @@
> > > +---
> > > +title: "R Markdown Super Awesome Homework 1"
> > > +author: "Izzy Gaw"
> > > +date: "2020-01-16"
> > > +output: html_document
> > > +---
> > > +
> > > +This is a text paragraph in an R Markdown document.
> > >
> > > Great job inserting your R code below. However, can you please replace
> the
> > > example text with your explanations?
> > > Remember, effective documentation is concise :)
> > > ------------------------------
> > >
> > > In Markdown_HW1_IzzyGaw.Rmd
> > > <
> https://github.com/Rbootcamp-UHM/Iris-repo/pull/13#discussion_r368255615>
> > > :
> > >
> > > > @@ -0,0 +1,20 @@
> > > +---
> > > +title: "R Markdown Super Awesome Homework 1"
> > > +author: "Izzy Gaw"
> > > +date: "2020-01-16"
> > > +output: html_document
> > > +---
> > > +
> > > +This is a text paragraph in an R Markdown document.
> > > +
> > > +Below is working R code in a code chunk:
> > >
> > > Just replace these lines with something informative
> > >
> > > —
> > > You are receiving this because you authored the thread.
> > > Reply to this email directly, view it on GitHub
> > > <
> https://github.com/Rbootcamp-UHM/Iris-repo/pull/13?email_source=notifications&email_token=AOI5PFU2IE5XQJEE6RED2T3Q6OKIVA5CNFSM4KIVXKRKYY3PNVWWK3TUL52HS4DFWFIHK3DMKJSXC5LFON2FEZLWNFSXPKTDN5WW2ZLOORPWSZGOCSH4YDQ#pullrequestreview-344968206
> >,
> > > or unsubscribe
> > > <
> https://github.com/notifications/unsubscribe-auth/AOI5PFS64FODZ33CNKKVKALQ6OKIVANCNFSM4KIVXKRA
> >
> > > .
> > >
> >
> >
> > --
> > *Isabella M. Gaw*
> > *Marine Biology **Master's Student*
> > *Cole Lab, **Department of Biology*
> > *University of Hawai'i at Mānoa*
> > —
> > You are receiving this because you commented.
> > Reply to this email directly, view it on GitHub, or unsubscribe.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Rbootcamp-UHM/Iris-repo/pull/13?email_source=notifications&email_token=AOI5PFUIGD4ZIBHRJGETCMLQ6OXANA5CNFSM4KIVXKRKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEJKGRVY#issuecomment-575957207>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AOI5PFXCRBWYJDG4KVVFHA3Q6OXANANCNFSM4KIVXKRA>
> .
>


-- 
*Isabella M. Gaw*
*Marine Biology **Master's Student*
*Cole Lab, **Department of Biology*
*University of Hawai'i at Mānoa*

mbutler808 commented:
Very nice job!!! Excellent Izzy! Marguerite
